docs/upgrade.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Upgrading
+layout: docs
+---
+
+You can find a list of Tilt releases on [Tilt's GitHub Releases page](https://github.com/windmilleng/tilt/releases).
```

Comment:
should we also put a link to this doc in the sidebar? or in the install guide?

Reply:
I kinda figured this page wasn't interesting enough to link to from the sidebar, and it was really only interesting as a landing page, but:

1. maybe it is actually useful?
2. in general, it's annoying when I land on a page and navigate away from it and try to navigate back, but it's not in the menu.

added